### ISSUE TITLE
YOLOv11 validation results has decreased in version 8.4.7

### ISSUE THREAD

**Issue by ou525** [ultralytics/ultralytics#23387]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

For the 8.4.7 version, when the YOLO11 model results are verified, the accuracy has decreased significantly. However, when I switch to the YOLO26 model that I trained myself, everything works properly, whether the end-to-end model is enabled or not.
model = YOLO('./runs/detect/train25/weights/best.pt')

# Validate the model
metrics = model.val(data='./data/13.yaml', end2end=False)

### Environment

Ultralytics 8.4.7 🚀 Python-3.11.9 torch-2.4.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3090 Ti, 24140MiB)
Setup complete ✅ (12 CPUs, 62.7 GB RAM, 284.5/442.1 GB disk)

OS                     Linux-5.15.0-165-generic-x86_64-with-glibc2.35
Environment            Docker
Python                 3.11.9
Install                git
Path                   /code/ultralytics2/ultralytics
RAM                    62.66 GB
Disk                   284.5/442.1 GB
CPU                    Intel Core(TM) i7-8700K 3.70GHz
CPU count              12
GPU                    NVIDIA GeForce RTX 3090 Ti, 24140MiB
GPU count              1
CUDA                   12.4

numpy                  ✅ 1.23.5>=1.23.0
matplotlib             ✅ 3.9.2>=3.3.0
opencv-python          ✅ 4.10.0.84>=4.6.0
pillow                 ✅ 10.2.0>=7.1.2
pyyaml                 ✅ 6.0.2>=5.3.1
requests               ✅ 2.32.3>=2.23.0
scipy                  ✅ 1.14.1>=1.4.1
torch                  ✅ 2.4.1+cu124>=1.8.0
torch                  ✅ 2.4.1+cu124!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision            ✅ 0.19.1+cu124>=0.9.0
tqdm                   ✅ 4.66.5>=4.64.0
psutil                 ✅ 6.0.0
py-cpuinfo             ✅ 9.0.0
pandas                 ✅ 2.2.3>=1.1.4
seaborn                ✅ 0.13.2>=0.11.0
ultralytics-thop       ✅ 2.0.9>=2.0.0

### Minimal Reproducible Example

model = YOLO('./runs/detect/train25/weights/best.pt')

# Validate the model
metrics = model.val(data='./data/13.yaml', end2end=False)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @ou525, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

This is an automated response 🤖—an Ultralytics engineer will also assist soon. In the meantime, thanks for the detailed environment info and for sharing your validation snippet 🙏

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. Your snippet is a good start, but to make it reproducible end-to-end, please include the smallest possible package of information needed to run and compare results (for example: exact model weights source, exact `data` yaml, command/config used to train, and a small sample of images/labels or a way to reproduce without private data).

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
I can't reproduce this with the pretrained YOLO11 model.

What version did you use to train your YOLO11 model?
